ginga/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = 'ginga' %}
-{% set version = '2.5.20160627100500' %}
-{% set rev = '25d6608b' %}
+{% set version = '2.5.20160706100500' %}
```

Comment:
@jhunkeler , I changed the date to yesterday since it is quite impossible for him to make a release back in time. Hope this works for you.